release.Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
im a bit confused, how is this different?

Reply:
Doesn't uninstall and rm foundry (mainly last two lines need to be removed from original docker file)

Reply:
can we just gate that by an arg instead of copying? it's kind of painful to maintain two copies -- i imagine someone will forget to bump both